src/AndroidDebugLauncher/InstallPaths.cs

```diff
@@ -121,6 +117,14 @@ out matchedPath
                 }
             }
 
+            IEnumerable<INDKFilePath> gdbServerPath = NDKPrebuiltFilePath.GDBServerPaths(targetArchitectureName);
+            result.GDBServerPath = GetNDKFilePath(
+                string.Concat("Android-", targetArchitectureName, "-GDBServerPath"),
+                ndkRoot,
+                gdbServerPath,
+                out matchedPath
+                );
```

Comment:
This is used when finding a possible gdb.exe. There are some if-statements using it.
It's not used in the case of gdbserver.

I can add a comment `// not used`.